libs/sdk-backend-tiger/src/backend/workspace/insights/index.ts

```diff
@@ -221,21 +221,18 @@ export class TigerWorkspaceInsights implements IWorkspaceInsightsService {
         const id = await objRefToIdentifier(ref, this.authCall);
 
         const apiResult = await this.authCall((client) =>
-            client.workspaceObjects.getEntityVisualizationObjects(
-                {
-                    objectId: id,
-                    workspaceId: this.workspace,
-                },
-                {
-                    headers: jsonApiHeaders,
-                    params: {
-                        include: "analyticalDashboards",
+            client.workspaceObjects.getAllEntitiesAnalyticalDashboards({
```

Comment:
probably need to get all pages here.. using the MetadataUtils.getAllpagesOf

Reply:
Good catch, I will update it